mk/spksrc.cross-meson-env.mk

```diff
@@ -84,21 +87,24 @@ $(MESON_CROSS_TOOLCHAIN_PKG):
 meson_pkg_toolchain: SHELL:=/bin/bash
 meson_pkg_toolchain:
 	@cat $(MESON_CROSS_TOOLCHAIN_WRK)
+	@echo "pkgconfig = '$$(which pkg-config)'"
```

Comment:
```suggestion
	@echo "pkgconfig = '$$(which pkgconfig)'"
```

Reply:
yes, but no.  There are other use-cases that requires `pkgconfig` or `pkg-config` definitions.  Including both solves that.